src/Microsoft.Diagnostics.Monitoring.WebApi/IExceptionsOperationFactory.cs

```diff
@@ -13,6 +14,6 @@ internal interface IExceptionsOperationFactory
         /// <summary>
         /// Creates an operation that produces an exceptions artifact.
         /// </summary>
-        IArtifactOperation Create(ExceptionFormat format);
+        IArtifactOperation Create(ExceptionFormat format, ExceptionsConfiguration configuration);
```

Comment:
This couples the artifact implementation to the HTTP API implementation. Rather than using the same type, please either create another "settings" type, pass each piece of data as separate parameters, etc; adding a parameter such as `IEnumerable<ExceptionFilter> include` would also "violate" this idea.

Reply:
Let me know if you would prefer different naming/folder locations for this (iirc it's currently ExceptionsConfigurationSettings)